src/reporters/json-reporter.js

```diff
@@ -26,8 +30,12 @@ export default class JSONReporter extends BaseReporter {
     this._dump('verbose', msg);
   }
 
-  list(type: string, items: Array<string>) {
-    this._dump('list', {type, items});
+  list(type: string, items: Array<string>, hints?: Object) {
+    if (hints) {
+      this._dump('list', {type, hints});
+    } else {
+      this._dump('list', {type, items});
```

Comment:
I said in a comment, remove the conditional and just make it `this._dump('list', {type, items, hints});`.